Sources/Errors/AlertCoordinator.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  AlertCoordinator.swift
+//  FuntastyKit
+//
+//  Created by Milan Strnad on 03/03/17.
+//  Copyright © 2017 The Funtasty s.r.o. All rights reserved.
+//
+
+import UIKit
+
+public final class AlertCoordinator {
+
+    let parentViewController: UIViewController
+    weak var viewController: UIAlertController?
+
+    var error: Error?
```

Comment:
Yep, that totally sounds reasonable. Will improve it before proceeding with merge.